test/behavior/builtin_functions_returning_void_or_noreturn.zig

```diff
@@ -22,7 +22,7 @@ test {
     try testing.expectEqual(noreturn, @TypeOf(if (true) @panic("") else {}));
     try testing.expectEqual({}, @prefetch(&val, .{}));
     try testing.expectEqual({}, @setAlignStack(16));
-    try testing.expectEqual({}, @setCold(true));
+    try testing.expectEqual({}, @branchHint(.cold));
```

Comment:
This doesn't trip `'@branchHint' must appear as the first statement in a function or conditional branch`?

Reply:
huh, yeah, that's... wrong. good spot